warehouse/models/mart/gtfs/fct_daily_scheduled_trips.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(materialized='table') }}
+
+WITH int_gtfs_schedule__daily_scheduled_service_index AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_schedule__daily_scheduled_service_index') }}
+),
+
+dim_trips AS (
+    SELECT *
+    FROM {{ ref('dim_trips') }}
+),
+
+dim_routes AS (
+    SELECT *
+    FROM {{ ref('dim_routes') }}
+),
+
+fct_daily_scheduled_trips AS (
+    SELECT
+        {{ dbt_utils.surrogate_key(['t1.service_date', 't2.key']) }} as key,
+        t1.service_date,
+        t1.feed_key,
+        t1.service_id,
+        t2.key AS trip_key,
+        t3.key AS route_key,
+        t2.warning_duplicate_primary_key AS warning_duplicate_trip_primary_key
+    FROM int_gtfs_schedule__daily_scheduled_service_index AS t1
+    LEFT JOIN dim_trips AS t2
+        ON t1.feed_key = t2.feed_key
+        AND t1.service_id = t2.service_id
+    LEFT JOIN dim_routes AS t3
```

Comment:
this is just a general style request, could aliases be made meaningful or just not use them? it's hard to read the `SELECT` and know where each column is actually coming from. I realize I haven't asked this until now.

Reply:
https://github.com/cal-itp/data-infra/pull/1936/commits/446c4290d2f02dea6d9f1e2eddfcde6212c1ae76